src/components/Footer.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <!-- Footer -->
+<footer class="page-footer font-small black-color" color="black">
+  <container class="text-left">
+      <row>
+        <col md="2">
+          <h5 class="text-uppercase mb-4 mt-3 font-weight-bold">AFRICAN INSTITUTE OF FINANCIAL MARKETS AND RISK MANAGEMENT</h5>
+          <p>Faculty of Commerce</p>
+          <p>University of Cape Town</p>
+        <col>
+      </row >
+  </container>
+
+  <!-- Copyright -->
+  <div class="footer-copyright text-center py-3">© 2021 Copyright: 
+    <a href="http://www.aifmrm.uct.ac.za/">AIFMRM</a>
+  </div>
+  <!-- Copyright -->
+
+</footer>
+  <!-- Footer -->
```

Comment:
why do you have these comments?

Reply:
It came from my tutorial so I thought they were important. Will adjust